### PR TITLE
Add release blog post for mongodb 1.4.0

### PR DESCRIPTION
**Pull request by alcaeus**

### CODE REVIEW

File: site/2016/11/22/doctrine-mongodb-release-1.4.0.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
+Doctrine MongoDB 1.4.0
+======================
+
+We are happy to announce the immediate availability of Doctrine MongoDB
+Abstraction Layer `1.4.0 <https://github.com/doctrine/mongodb/releases/tag/1.4.0>`__.
+
+Passing context options to the driver
+-------------------------------------
+
+With this release it's possible to driver options to the connection class, which
```
Comment (mikeSimonson) [inline site/2016/11/22/doctrine-mongodb-release-1.4.0.rst]: it's possible to pass I guess ?

Comment (alcaeus) [inline site/2016/11/22/doctrine-mongodb-release-1.4.0.rst]: Yes, thank you!

File: site/2016/11/22/doctrine-mongodb-release-1.4.0.rst [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        ]
+    ]);
+    $connection = new \Doctrine\MongoDB\Connection(null, [], null, null, ['context' => $context]);
+
+Passing multiple expressions to logical operators
+-------------------------------------------------
+
+The ``addAnd``, ``addNor`` and ``addOr`` methods in the query and aggregation
+builders now take multiple expression objects. Instead of having to call the
+method repeatedly, you need to only call it once now:
```
Comment (jmikola) [inline site/2016/11/22/doctrine-mongodb-release-1.4.0.rst]: "you may call it once with multiple arguments". I suppose the old style still works, but saying "you need to call it once now" made it sound like we're breaking BC.

File: site/2016/11/22/doctrine-mongodb-release-1.4.0.rst [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+ODM prepare for the next version of ODM which will utilize the new MongoDB
+library API.
+
+Bug fixes in this release
+-------------------------
+
+Notable fixes may be found in the
+`changelog <https://github.com/doctrine/mongodb/blob/master/CHANGELOG-1.4.md#140-2016-11-22>`__.
+A full list of issues and pull requests included in this release may be found in the
+`1.4.x milestone <https://github.com/doctrine/mongodb/issues?q=milestone%3A1.4.x>`__.
```
Comment (jmikola) [inline site/2016/11/22/doctrine-mongodb-release-1.4.0.rst]: Is there a specific 1.4.0 milestone? I would expect 1.4.x to be for things that didn't make it to 1.4.0.

File: site/2016/11/22/doctrine-mongodb-release-1.4.0.rst [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+With this release, we have dropped support for PHP 5.6. Users using PHP 5.5 or
+older are encouraged to upgrade to a newer PHP version. If you are using PHP 7.0
+or 7.1, you can use this library by adding a polyfill for ``ext-mongo``, like
+`mongo-php-adapter <https://github.com/alcaeus/mongo-php-adapter>`__.
+
+Future releases
+---------------
+
+This release is the last planned release of the MongoDB Abstraction Layer. The
```
Comment (jmikola) [inline site/2016/11/22/doctrine-mongodb-release-1.4.0.rst]: "last planned minor release". May want to clarify that this means no 1.5, but 1.4.x maintenance releases may still happen.

**Pull request merged**